### PR TITLE
ct-form-layout rework & system overview form improvements

### PR DESCRIPTION
**Pull request by garrett**

### CODE REVIEW

**Comment by garrett** [comment]
I'll drop the form-layout code for easier merging.

File: pkg/systemd/host.css [hunk 1]
```diff
@@ -278,6 +278,12 @@ body {
     word-wrap: break-word;
 }
 
+#system_machine_id {
+    display: inline-block;
+    font-family: monospace;
+    word-break: break-all;
```
Comment (martinpitt) [inline pkg/systemd/host.css:284]: Is word-wrapping really intended? I thought we wanted to ellipsize and show a tooltip, or not wrap and just make the  box wide enough and drop the tooltip?

Comment (martinpitt) [reply pkg/systemd/host.css:284]: Answering myself: the wrapping makes sense for really narrow displays, and it looks ok. I'll drop the tooltip.

**Comment by martinpitt** [comment]
Incorporated into https://github.com/cockpit-project/cockpit/pull/10974 with dropping the machine ID tooltip and some squashing. Thank you!

**Pull request closed without merging**